### PR TITLE
forward packaged commands test outputs to console

### PR DESCRIPTION
**Pull request by anurse**
This should help diagnose failures in these tests.

/cc @pakrym @davidfowl @piotrpMSFT

### CODE REVIEW

File: scripts/dotnet-cli-build/TestTargets.cs [hunk 1]
```diff
@@ -163,7 +163,12 @@ public static BuildTargetResult RunPackageCommandTests(BuildTargetContext c)
             // Test the apps
             foreach (var dir in Directory.EnumerateDirectories(consumers))
             {
-                var result = dotnet.Exec("hello").WorkingDirectory(dir).CaptureStdOut().CaptureStdErr().Execute();
+                var result = dotnet.Exec("hello").WorkingDirectory(dir)
+                    .ForwardStdErr()
+                    .ForwardStdOut()
+                    .CaptureStdOut()
```
Comment (pakrym) [inline scripts/dotnet-cli-build/TestTargets.cs]: Why capture and forward?

Comment (anurse) [inline scripts/dotnet-cli-build/TestTargets.cs]: We need to check the output further down. So we want to do both here.

**Comment by pakrym** [comment]
Didn't notice checking the output :shipit: after green build.

**Comment by piotrpMSFT** [comment]
@anurse merging this as it'll help unblock CI.

**Pull request merged**